prometheus-ksonnet/lib/prometheus.libsonnet

```diff
@@ -75,7 +75,10 @@
     local volumeMount = $.core.v1.volumeMount,
 
     prometheus_config_mount::
-      $.util.configVolumeMount('%s-config' % self.name, _config.prometheus_config_dir),
+      $.util.configVolumeMount('%s-config' % self.name, _config.prometheus_config_dir)
+      + $.util.configVolumeMount('%s-alerts' % self.name, _config.prometheus_config_dir + '/alerts')
+      + $.util.configVolumeMount('%s-rules' % self.name, _config.prometheus_config_dir + '/recording')
```

Comment:
Shouldn't this be `/rules` is in the HA setup above?

Reply:
yes!